strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -92,6 +93,30 @@
 
     private DateFormat formatter = new SimpleDateFormat("yyyyMMdd.HHmmss");
 
+    private static final String REPO_PREFIX = "amsi-releases-without-deployment";
```

Comment:
@yugander-krishan-singh ,

* According to @sbespalov, you don't need to define unique repository names anymore. You can drop all the prefixes.
* Could you please rename the repositories below to ones that follow this naming convetion:
  * `RELEASES_WITHOUT_REDEPLOYMENT` for `amsi-releases-without-redeployment`

Reply:
Sure @carlspring. I will make the changes.